components-core/src/main/java/com/adyen/checkout/components/core/PaymentComponentData.kt

```diff
@@ -37,6 +37,7 @@ data class PaymentComponentData<PaymentMethodDetailsT : PaymentMethodDetails>(
     var dateOfBirth: String? = null,
     var socialSecurityNumber: String? = null,
     var installments: Installments? = null,
+    var supportNativeRedirect: Boolean? = true,
```

Comment:
Leaving a comment so we don't forget to check if this should be changed to `false`